webcompat/webhooks/helpers.py

```diff
@@ -17,6 +17,8 @@
 from webcompat.helpers import extract_url
 from webcompat.helpers import proxy_request
 
+BROWSERS = ['blackberry', 'brave', 'chrome', 'edge', 'firefox', 'iceweasel', 'ie', 'lynx', 'myie', 'opera', 'puffin', 'qq', 'safari', 'samsung', 'seamonkey', 'uc', 'vivaldi']  # noqa
```

Comment:
So the idea is, "these are the browsers we know about, we'll manually label anything else"?

Reply:
I like your solution -- I think it covers the 99% use-cases just fine.